server/lib/parties/party-api.ts

```diff
@@ -58,15 +60,15 @@ function convertPartyServiceError(err: Error) {
     case PartyServiceErrorCode.NotFoundOrNotInvited:
     case PartyServiceErrorCode.NotFoundOrNotInParty:
     case PartyServiceErrorCode.InvalidAction:
-      throw new httpErrors.BadRequest(err.message)
+      throw new HttpErrorWithPayload(400, err.message, { code: err.code })
```

Comment:
Small thing, but feel like we could probably write a helper to make this less repetitive, pass it an HTTP error code + a `Error & { code: string }` and it makes this call?

`asHttpError(400, err)` feels a bit easier to parse to me, anyway